### PR TITLE
[pulsar-client-tools] add tls truststore and hn-verification option to argument

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation
Allow configure `tlsTrustCertsFilePath` `tlsAllowInsecureConnection` `tlsEnableHostnameVerification` using CLI argument in pulsar-client tools.

### CODE REVIEW

**Review by jiazhai — DISMISSED** [review]

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java [hunk 1]
```diff
@@ -53,13 +53,18 @@
             "or \"{\"key1\":\"val1\",\"key2\":\"val2\"}.")
     String authParams = null;
 
+    @Parameter(names = { "--tls-trust-cert-path" }, description = "Allow TLS trust cert file path")
+    String tlsTrustCertsFilePath;
+
+    @Parameter(names = { "--tls-allow-insecure" }, description = "Allow TLS insecure connection")
+    Boolean tlsAllowInsecureConnection = null;
```
Comment (sijie) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java:60]: This is a boxed value. 

I think you need to change the value assignments in PulsarClientTool constructor.

```
        this.tlsAllowInsecureConnection = Boolean
                .parseBoolean(properties.getProperty("tlsAllowInsecureConnection", "false"));
        this.tlsEnableHostnameVerification = Boolean
                .parseBoolean(properties.getProperty("tlsEnableHostnameVerification", "false"));
        this.tlsTrustCertsFilePath = properties.getProperty("tlsTrustCertsFilePath");
```

**Comment by codelipenghui** [comment]
@rdhabalia Could you please take a look at sijie's comments?

**Comment by codelipenghui** [comment]
move to 2.7.0 first.

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java (hunk 1 above)
Comment (rdhabalia) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java:60]: yes, that's correct. I fixed it.

**Comment by tisonkun** [comment]
Closed as stale and conflict. Please rebase and resubmit the patch if it's still relevant.

**Pull request closed without merging**